custom/languages/en_UK.json

```diff
@@ -189,6 +189,11 @@
     "admin/emails_mass_message": "Email Mass Message",
     "admin/emails_mass_message_loading": "Loading... Please do not reload the page. This may take a while.",
     "admin/emails_mass_message_replacements": "You can use variables in your email message. Supported variables: {username}, {sitename}",
+    "admin/emoji_style": "Emoji Style",
+    "admin/emoji_style_help": "Change which emoji set is used:<br>&bull; Native {{nativeExample}} <br>&bull; Twemoji {{twemojiExample}} <br>&bull; JoyPixels {{joypixelsExample}}",
+    "admin/emoji_native": "Native",
```

Comment:
Wondering if this will translate well into other languages. Should it maybe be "OS Native"? Unsure

Reply:
It's fine, the translation doesn't necessarily have to be a literal translation of the source string